soda/core/soda/common/json_helper.py

```diff
@@ -68,6 +69,8 @@ def to_jsonnable(o):
             return o.value
         if isinstance(o, Undefined):
             return None
+        if isinstance(o, Exception):
```

Comment:
What caused having to implement this?

Reply:
I'm surprised that this comes up now since I believe soda-core already has a bunch of exceptions as part of their logs no?

Reply:
For posterity as discussed in DMs: this didn't come up before because all usages of logs either stringify the exception or perform string completion.

I would still be in favour of this handling though since it's a more generic/safe option in case developers forget to stringify things before.

Reply:
LGTM